src/lang/type.ml

```diff
@@ -459,6 +476,11 @@ let print_repr f t =
     | `Range_Ellipsis ->
         Format.fprintf f "...";
         vars
+    | `Misc (a, b, c) ->
```

Comment:
I would suggest a more explicit name. It seems that this is only used for debugging so perhaps `` `Debug``